src/ResourceManager/SiteRecovery/Commands.SiteRecovery/help/Edit-AzureRmSiteRecoveryRecoveryPlan.md

```diff
@@ -143,7 +145,7 @@ Accept wildcard characters: False
 ```
 
 ### -RemoveProtectedEntities
-Specifies an array of protected entities.
+An instance of an ASRProtectionEntity array of protected entities to remove.
```

Comment:
An instance of an array seems confusing. This is an array of ASRProtectionEntity instances to remove